### PR TITLE
update writeDump.json per #991

### LINKED ISSUE #991: Syntax for writeDump shows Lucee ordering not ACF
For syntax we are showing the argument order for lucee, which is incompatible with Lucee. A bug has been filed with lucee to hopefully match ACF: https://luceeserver.atlassian.net/browse/LDEV-2045 (checkout the trycf examples in that bug report to see the issue).

### PR DESCRIPTION
**Pull request by atuttle**
Resolves #991.

In addition to re-ordering the arguments to match the ACF order, I added a note to the Lucee notes box referencing the issue linked from #991.

### CODE REVIEW

**Comment by KamasamaK** [comment]
There is an issue with the commas in `params`.

File: data/en/writedump.json [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 		{"name":"abort","description":"Boolean value to immediately abort after displaying the dump.","required": false,"default":false,"type":"boolean","values":[true,false]},
-		{"name":"hide","description":"hide column or keys.","required":false,"default":"","type":"string","values":[]},
-		{"name":"keys","description":"For a structure, number of keys to display.","required":false,"default":"","type":"numeric","values":[]},
 		{"name":"label","description":"A string; header for the dump output.","required":false,"default":"","type":"string","values":[]},
 		{"name":"metainfo","description":"Includes information about the query in the cfdump results.","required":false,"default":true,"type":"boolean","values":[true,false]},
-		{"name":"output","description":"Where to send the results of cfdump.","required":false,"default":"browser","type":"string","values":["browser","console","filename"]},
+		{"name":"top","description":"The number of rows to display.","required":false,"default":"","type":"numeric","values":[]}
```
Comment (pfreitag) [inline data/en/writedump.json]: Missing a trailing comma on this line

File: data/en/writedump.json [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 		{"name":"show","description":"show column or keys.","required":false,"default":"","type":"string","values":[]},
+		{"name":"hide","description":"hide column or keys.","required":false,"default":"","type":"string","values":[]},
+		{"name":"keys","description":"For a structure, number of keys to display.","required":false,"default":"","type":"numeric","values":[]},
+		{"name":"expand","description":"Yes: In Internet Explorer and Mozilla, expands views","required":false,"default":true,"type":"boolean","values":[true,false]},
 		{"name":"showUDfs","description":"show UDFs in cfdump output.","required":false,"default":true,"type":"boolean","values":[true,false]},
```
Comment (pfreitag) [inline data/en/writedump.json]: This trailing comma needs to be removed since it is the last element of the array.

**Comment by atuttle** [comment]
Sorry about that. I had to disable Prettier to prevent it from completely re-spacing the entire file, and I've come to rely on it to fix problems like that for me. :)

**Pull request merged**